packages/blade/src/components/Breadcrumb/_decisions/decisions.md

```diff
@@ -0,0 +1,206 @@
+# Breadcrumb Decisions <!-- omit in toc -->
+
+Breadcrumbs are used for navigating through or to show a user’s location in an application.
+
+<img width="426" alt="Breadcrumb Thumbnail" src="./breadcrumb-thumbnail.png" />
+
+- [Design](#design)
+- [Anatomy](#anatomy)
+- [Basic Usage](#basic-usage)
+- [API](#api)
+  - [Breadcrumb](#breadcrumb)
+  - [BreadcrumbItem](#breadcrumbitem)
+  - [Examples](#examples)
+    - [Show Last Separator](#show-last-separator)
+  - [Usage with Routing Libraries](#usage-with-routing-libraries)
+- [Accessibility](#accessibility)
+- [Open Questions](#open-questions)
+- [References](#references)
+
+## Design
+
+[Figma Link](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade-DSL?type=design&node-id=81454-67196&mode=design&t=6sCjPO2iUqIMuPHT-0) to all variants of the Breadcrumb component
+
+## Anatomy
+
+<img src="./breadcrumb-anatomy.png" width="100%" alt="Anatomy of breadcrumbs" />
+
+## Basic Usage
+
+```jsx
+<Breadcrumb size="small" color="primary">
+  <BreadcrumbItem href="/">Home</BreadcrumbItem>
+  <BreadcrumbItem href="/dashboard">Dashboard</BreadcrumbItem>
+  <BreadcrumbItem href="/settlements" isCurrentPage>Settlements</BreadcrumbItem>
+</Breadcrumb>
+```
+
+## API
+
+### Breadcrumb
+
+```ts
+type BreadcrumbProps = StyledPropsBlade & {
+  /**
+   * Size of the Breadcrumb
+   * 
+   * @default medium
+   */
+  size?: 'small' | 'medium' | 'large';
+  /**
+   * Color of the Breadcrumb
+   * 
+   * @default neutral
+   */
+  color?: 'neutral' | 'primary' | 'white';
+  /**
+   * Content of the Breadcrumb, accepts BreadcrumbItem
+   */
+  children: React.ReactNode;
+  /**
+   * Whether to show the last separator
+   */
+  showLastSeparator?: boolean;
+};
+```
+
+### BreadcrumbItem
+
+```ts
+type BreadcrumbItemProps = {
+  /**
+   * Href of the BreadcrumbItem
+   */
+  href: string;
+  /**
+   * Function to be called on click of the BreadcrumbItem,
+   * 
+   * This can be used to integrate with routing libraries like `react-router-dom`
+   */
+  onClick?: (event: React.MouseEvent<HTMLAnchorElement>) => void;
+  /**
+   * Whether the BreadcrumbItem is the current page,
+   * Sets the aria-current attribute to `page`
+   * 
+   * @default false
+   */
+  isCurrentPage?: boolean;
+  /**
+   * Content of the BreadcrumbItem
+   */
+  children: string;
```

Comment:
According to the design there can be cases where the first item shows only an icon. Can we make this optional in case `icon` prop is supplied?

Reply:
Right, this will be optional. I'll change this.